pom.xml

```diff
@@ -476,6 +476,36 @@
             <plugins.docker>docker</plugins.docker>
          </properties>
       </profile>
+      
+      <profile>
+         <id>site</id>
+         <activation>
+            <property>
+               <name>make-site</name>
+            </property>
```

Comment:
Just so no one else misses this. `-Pmake-site` doesn't work, and `-Dmake-site` does work